enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/HighAvailabilityMemberStateMachine.java

```diff
@@ -235,16 +235,49 @@ public void notify( HighAvailabilityMemberListener listener )
             }
         }
 
+
+        /**
+         * As soon as we receive unavailability message and instanceId belongs us we depending from currentState
```

Comment:
The end of this sentence was kind of hard to understand. Suggested rewrite:
`As soon as we receive an unavailability message and the instanceId belongs to us, depending on the current state we do the following:`